general/src/protocol/user_feedback.rs

```diff
@@ -0,0 +1,18 @@
+use crate::protocol::EventId;
+
+use serde::{Deserialize, Serialize};
+
+/// User feedback for an event as sent by the client to the userfeedback/userreport endpoint.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct UserFeedback {
```

Comment:
@untitaker this is called `UserReport` in Sentry, let's align with this terminology, please.